Components/grid.css

```diff
@@ -0,0 +1,25 @@
+.grid {
+    display: grid;
+    padding: 5px;
+}
+
+.col-2 {
+    grid-template-columns: auto auto;
+  }
+  
+.col-3 {
+    grid-template-columns: auto auto auto;
```

Comment:
this is also correct , you can also try repeat(3,1fr) (just a suggestion )

Reply:
Thanks for suggestion  , I will implement it.